src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareUtils.cs

```diff
@@ -23,31 +25,62 @@ namespace Microsoft.SqlTools.ServiceLayer.SchemaCompare
     /// </summary>
     internal static class SchemaCompareUtils
     {
+        /// <summary>
+        /// BooleanOptionsDictionary has all the deployment options and is being processed separately in the second iteration by collecting it in first iteration.
```

Comment:
This comment is a little confusing. Specific implementation details like this can be in the body of the method itself, next to the related code. For the method summary comment, this can just say that this converts DeploymentOptions used in STS and ADS to DacDeployOptions which can be passed to the in the DacFx apis.

Reply:
I'll update it here and at below code, Benjin requested to add somewhere how we are handling these boolean and non-boolean options, so this is the place where we prepare the options to DacFx, and thought it would be nice to add here. Anyway the code it self has the comment explaining it.